src/Resources/app/storefront/src/checkout/confirm-order.plugin.js

```diff
@@ -16,12 +16,11 @@ export default class ConfirmOrderPlugin extends Plugin {
             locale,
             clientKey,
             environment,
-            showPayButton: false,
+            showPayButton: true,
```

Comment:
So this is affecting all PMs now?

Reply:
this is only for the AdyenCheckout instance in the confirm page itself, it only renders the payment buttons and handles additionalActions, and actually the `showPayButton` value is already overwritten in the payment button config used to create the button, I am just moving it up into the AdyenCheckout object. There is a separate AdyenCheckout object declared in `checkout.plugin.js` which handles the other PMs